assert/assertions.go

```diff
@@ -217,8 +215,10 @@ func CallerInfo() []string {
 	var line int
 	var name string
 
-	callers := []string{}
+	const stackFrameBufferSize = 10
```

Comment:
Maybe a comment could be added mentioning why 10 was chosen.

Reply:
10 is the value in the example from the [`runtime.Frames` documentation](https://pkg.go.dev/runtime#Frames).